sphinx_gallery/scrapers.py

```diff
@@ -492,6 +499,13 @@ def _single_line_sanitize(s):
    :class: sphx-glr-single-img
 """
 
+# Used for SVGs
+SVG_IMAGE = """
+.. image:: /%s
+   :alt: %s
+   :class: sphx-glr-single-img
+"""
```

Comment:
Except for that initial space. Since it said it was kept for backwards compatibility I wasn't sure if the space should still be there to not break anything. But maybe the space is always an issue?

Reply:
Yeah I think the space is a bug and can safely be removed